app/src/main/java/com/example/benchproject/app/common/navigator/NavigatorForFragment.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.benchproject.app.common.navigator
+
+import androidx.fragment.app.Fragment
+import com.google.android.material.snackbar.Snackbar
+
+abstract class NavigatorForFragment {
```

Comment:
`FragmentNavigator` wouldn't be anough?

Reply:
![image](https://user-images.githubusercontent.com/57491794/175915491-af5a962a-28bd-4ee9-ae1c-6e6a90f70d7f.png)
### I would like to make different name than this in the android library if it is not a bad idea I can name this like `FragmentNavigator`

Reply:
I changed name `NavigatorForFragment` -> `FragmentNavigator`